Production/test/condorSub/dict_Autumn18Fast_pmssm.py

```diff
@@ -0,0 +1,11 @@
+flist = {
+    "scenario": "Autumn18Fastsig",
```

Comment:
Wouldn't you want an args section in these dictionaries to turn on things like QGLikelihood or are there no arguments needed?

Reply:
The V17 production does not use args in dictionaries.